android/libraries/rib-base/src/test/java/com/badoo/ribs/core/RouterTest.kt

```diff
@@ -66,7 +66,7 @@ class RouterTest {
     @Test
     fun `Permanent parts are attached on attach`() {
         router.onAttach(null)
-        verify(node).attachChild(childNode1, null)
-        verify(node).attachChild(childNode2, null)
+        verify(node).attachChildNode(childNode1, null)
```

Comment:
Also checked this class, and looks like there no tests that whenever we are using methods:
`push/popBackStack` etc. `backStackManager` invoked.
From one point of view it is simple proxy, but let's discuss on RibComittie should we cover it with Tests or not.

Reply:
Yes, thanks.